src/internal/translations/en-US.json

```diff
@@ -164,6 +164,31 @@
         "comment": "",
         "limit": 0
     },
+    "dashboard.bucket.location_title.pushpin": {
+        "value": "Location",
+        "comment": "Geological location (position of a pin pushed to a map).",
+        "limit": 0
+    },
+    "dashboard.bucket.size_title.pushpin": {
+        "value": "Measure",
+        "comment": "",
+        "limit": 0
+    },
+    "dashboard.bucket.size_subtitle.pushpin": {
+        "value": "Size",
+        "comment": "Geological size (size of a pin pushed to a map).",
+        "limit": 0
+    },
+    "dashboard.bucket.color_title.pushpin": {
+        "value": "Measure",
+        "comment": "",
+        "limit": 0
+    },
+    "dashboard.bucket.color_subtitle.pushpin": {
+        "value": "Color",
+        "comment": "Geological color (color of a pin pushed to a map).",
```

Comment:
Please use only "Color of a pin pushed to a map" because "Geological size" doesn't make sense.

Reply:
fixed in [1368](https://github.com/gooddata/gooddata-react-components/pull/1368)